omegaup/validator/validatortest.py

```diff
@@ -51,7 +51,45 @@ def run(
         return result
 
 
-def main() -> None:
+class TestRunnerV2(unittest.TextTestRunner):
+    """A unittest.TestRunner that prints 1 on standard output on success.
+    If the associated .out file has contents, the test will consider
+    it to be the expected failure message for the case and will fail
+    with JE instead of WA. This is useful for invalid-inputs-enabled
+    CI runs."""
+
+    def run(self,
+            test: Union[unittest.suite.TestSuite, unittest.case.TestCase]
+            ) -> unittest.result.TestResult:
```

Comment:
```suggestion
    def run(
            self,
            test: Union[unittest.suite.TestSuite, unittest.case.TestCase],
    ) -> unittest.result.TestResult:
```

por consistencia con L45